### DISCUSSION TITLE
Q: 更新版本后出现页面不加载/白屏 || Q: After updating the version, the page does not load/white screen

### DISCUSSION THREAD

**Discussion by XiHua0522** [walinejs/waline#2060] [Q&A] [answered]
### Describe the bug | 问题描述

![image](https://github.com/walinejs/waline/assets/98436743/f3c46bf0-83f3-4f79-9d7e-f2ad9b7b6956)
![60f10e9cdc086bd21f244d084e54edc](https://github.com/walinejs/waline/assets/98436743/dd6999a5-907c-4ea4-b675-4ff96d2792b0)
就像这样，什么都没有

### Website URL | 问题网站

https://waline.xihua.shop/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

![image](https://github.com/walinejs/waline/assets/98436743/f3c46bf0-83f3-4f79-9d7e-f2ad9b7b6956)
![60f10e9cdc086bd21f244d084e54edc](https://github.com/walinejs/waline/assets/98436743/dd6999a5-907c-4ea4-b675-4ff96d2792b0)
like this, nothing

### Website URL | Problem Website

https://waline.xihua.shop/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
看起来是 unpkg.com 的网络问题导致的资源加载失败。这个页面并不影响你的网站使用，可以忽略。

---
It looks like a network issue with unpkg.com is causing resource loading failures. This page does not affect your website use and can be ignored.

**Reply by XiHua0522** [reply]
解决了，使用代理软件即可正常访问，感谢！

---
Solved, use proxy software to access normally, thanks!

**Reply by Benn314** [reply]
请问使用代理软件要怎么解决，我也遇到此问题，但仍无法解决😢

---
May I ask how to solve it using proxy software, I also encountered this problem, but still can't solve it 😢

**Reply by XiHua0522** [reply]
试过把代理软件设置为全局代理吗，正常来说出现这问题应该跟你网络环境有关，可能换个网就没问题了

---
Have you tried setting the proxy software as a global proxy? Normally, this problem should be related to your network environment. It may be fine if you change the network

**Reply by Benn314** [reply]
可以了！看来是我谷歌浏览器插件的VPN伪全局，我用clash本地设置全局就可以了，谢谢你！🥳

---
OK! It seems to be the VPN pseudo-global of my Google browser plug-in. I can use Clash to set the global locally, thank you! 🥳

**Comment by appotry** [comment]
新版本遇到同样的问题，刷新几下，然后就好了

---
The new version encounters the same problem, refresh a few times, and then it will be fine

**Reply by Benn314** [reply]
试过了，没用😢

---
Tried it, didn't work 😢

**Reply by appotry** [reply]
unpkg.com 把这个域名加入代理

---
unpkg.com add this domain name to agent
